example/Demo.vue

```diff
@@ -73,24 +72,48 @@
 
     <div class="example">
       <h3>Format datepicker</h3>
-      <Datepicker :format="format" />
-      <code>&lt;datepicker :format="format"&gt;&lt;/datepicker&gt;</code>
+      <Datepicker :format="format" v-model="state"/>
```

Comment:
The model isn't needed to show the format usage

Reply:
OK - I've done that.